lib/plugins/common/cctx.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash/fp')

```

Comment:
is the file supposed to be called cctx instead of ccxt?

Reply:
No, it's not, good catch!